### ISSUE TITLE
Submit to homebrew upstream or change repo format to tappable format?

### ISSUE THREAD

**Issue by colindean** [martido/brew-graph#3]
I don't know if the Homebrew team might be willing to pull this in as the new brew-graph command, since it was removed in Homebrew/homebrew@2496da48b34ee183b6873e090e91b0adf87cb674.

Otherwise, it'd be awesome if this repo turned into a usable tap, by moving the command to the `cmd` directory and changing the name to `homebrew-brew-graph` or something like that.

**Comment by michaelblyons** [comment]
FWIW, I find the `fdp` command to produce more readable graphs than `dot` for non-trivial Homebrew package lists.

**Comment by colindean** [comment]
@michaelblyons Can you expand on that, perhaps with examples?

**Comment by michaelblyons** [comment]
The README suggests this code for sample usage, which makes a graph like [your comment above](https://github.com/martido/brew-graph/issues/3#issuecomment-64039129):

``` sh
brew install graphviz
brew graph --installed | dot -Tpng -ograph.png
open graph.png
```

But I find those snake-y lines to be hard to follow for a dependency tree that large. Instead, using `fdp` makes me a bubble map that, while not perfect, is easier for me to see relationships.

``` sh
brew graph --installed | dot -Tpng -ograph.png
```

Perhaps this was the wrong thread to remark upon it. :)

**Comment by colindean** [comment]
I think you pasted the same thing twice, accidentally, @michaelblyons.

**Comment by michaelblyons** [comment]
> I think you pasted the same thing twice, accidentally, @michaelblyons.

So I did. :) Edited.

**Comment by martido** [comment]
Hi @michaelblyons, I've personally never used fdp, but the graph looks nice indeed. I think I'm going to add it to the README as an alternative, incl. some sample graphs.

**Comment by colindean** [comment]
Holy crap, `fdp` is infinitely more readable than dot for my output.

`fdp`:
![graph](https://user-images.githubusercontent.com/197224/50524396-67dd1580-0aa3-11e9-8d57-be8e67074634.png)

`dot`:
![dgraph](https://user-images.githubusercontent.com/197224/50524397-6c093300-0aa3-11e9-94c9-8a61779c49e1.png)

**Comment by martido** [comment]
I've recently had a bit more time again to work on brew-graph. I will close this issue because there's been a tap repository for quite some time now. I think having brew-graph as an external command is the way to go instead of trying to get it into Homebrew itself. However, according to [the documentation](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap), the current naming convention of homebrew-brew-graph is not correct. It should be homebrew-graph if I understand it correctly. I will look into this and maybe change a few things to adhere more to the standards.

Btw, I've added an example of using fdp to the README.
